cocos/2d/CCMotionStreak.h

```diff
@@ -86,6 +86,16 @@ class CC_DLL MotionStreak : public Node, public TextureProtocol
      * @param bFastMode True if enabled fast mode.
      */
     inline void setFastMode(bool bFastMode) { _fastMode = bFastMode; }
+    /** Get stroke.
+     *
+     * @return float stroke.
+     */
+    inline float getStroke() const { return _stroke; }
+    /** Set stroke.
+     *
+     * @param float stroke.
```

Comment:
It is a wrong doxygen comment.
I think the correct way is 

@param stroke description_of_stroke